src/components/DateRangeSelector.tsx

```diff
@@ -0,0 +1,61 @@
+// DateRangeSelector.tsx
+
+// Before Using Install the required dependencies
+/*npm install react-datepicker
+npm install date-fns*/
```

Comment:
These changes can be added to the package.json. Rather than manually installing these dependencies each time, a new dev should only need to run `npm install` to get everything they need.